### DISCUSSION TITLE
New Plugin for Microsoft Storage Space

### DISCUSSION THREAD

**Discussion by alex6dj** [Rem0o/FanControl.Releases#708] [Show and tell]
Hello, it's been a year since I started using this excellent program. One problem that I had from the first moment is that the driver it uses (LibreHardwareMonitor) does not provide information on the temperature of the hard drives if they had been previously added to a Microsoft Storage Space. 

The problem is that this information can only be obtained through Powershell. So I developed a small plugin that allows you to monitor the hard drives that have been added to a Storage Space and thus control the fans near them.

Without further ado, here is the link for anyone who is interested.

[FanControl.StorageSpacePlugin](https://github.com/QAppsSoft/FanControl.StorageSpacePlugin)

**Comment by AtA3301** [comment]
> The problem is that this information can only be obtained through Powershell. So I developed a small plugin that allows you to monitor the hard drives that have been added to a Storage Space and thus control the fans near them.
> 
> Without further ado, here is the link for anyone who is interested.
> 
> [FanControl.StorageSpacePlugin](https://github.com/QAppsSoft/FanControl.StorageSpacePlugin)

Nice one!
